docs/design/design.md

```diff
@@ -84,7 +84,7 @@ At this point users can test Kubernetes by using the exported kubeconfig.
 
 ### Cluster Deletion
 
-All "node" containers in the cluster are tagged with docker labels identifying
+All "node" containers in the cluster are tagged with container labels identifying
```

Comment:
again, this is searchable, I'm not sure I want to sacrifice this. docker labels have a lot of restrictions and particular behavior, what those are is probably interesting to many of our users / developers who may not have used these yet.